src/components/branding/branding.constant.ts

```diff
@@ -42,13 +42,14 @@ export const jsonBranding = JSON.stringify({
   'docs': {
     'devfile': 'https://www.eclipse.org/che/docs/che-7/making-a-workspace-portable-using-a-devfile/',
     'workspace': 'https://www.eclipse.org/che/docs/che-7/workspaces-overview/',
-    'factory': 'https://www.eclipse.org/che/docs/factories-getting-started.html',
-    'organization': 'https://www.eclipse.org/che/docs/organizations.html',
-    'converting': 'https://www.eclipse.org/che/docs/che-7/converting-a-che-6-workspace-to-a-che-7-devfile/',
+    'factory': 'https://www.eclipse.org/che/docs/che-7',
```

Comment:
This factory reference is not visible by default because it's used in the hidden UI, right?

Reply:
yes, correct